iamf/cli/proto/codec_config.proto

```diff
@@ -41,6 +41,9 @@ message OpusEncoderMetadata {
   optional OpusApplicationFlag application = 2;
```

Comment:
Do you imagine any use cases where `application` should vary per substream (or per audio element)?

I'm thinking of a use case where one audio element is music (best encoded as `APPLICATION_AUDIO`) and a second audio element is voice chat (best encoded as `APPLICATION_VOID`).

If you think this would be useful, we may want to consider how we would handle this - even if it does not get implemented in this PR.

Reply:
I personally don't see this as a critical addition right now; I'd think separate audio elements would be used in that case. But yeah, the need may materialize suddenly at some point.

Reply:
I personally don't see this as a critical addition right now; I'd think separate audio elements would be used in that case. But yeah, the need may materialize suddenly at some point.